apps/coordinator/src/components/Wallet/TransactionAnalysis/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Typography,
+  Box,
+} from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { SWASlider } from "./SWASlider";
+
+export const TransactionAnalysis = () => {
+  return (
+    <Box className="transaction-analysis-container">
+      <Accordion defaultExpanded>
```

Comment:
This is a pretty complex tool and i think it's probably better to keep it closed to start. 
```suggestion
      <Accordion>
```